pep_sphinx_extensions/pep_zero_generator/subindices.py

```diff
@@ -43,8 +46,26 @@ def generate_subindicies(
 This is the index of all Python Enhancement Proposals (PEPs) labelled
 under the '{subindex.title()}' topic. This is a sub-index of :pep:`0`,
 the PEP index.
+
+{additional_description}
 """
         subindex_text = writer.PEPZeroWriter().write_pep0(
-            filtered_peps, header, subindex_intro, is_subindex=True,
+            filtered_peps, header, subindex_intro, is_pep0=False,
         )
         update_sphinx(f"topic/{subindex}", subindex_text, docnames, env)
+
+
+def generate_topic_contents(docnames: list[str], env: BuildEnvironment):
+    update_sphinx(f"topic/index", """\
+Topic Index
+***********
+
+PEPs are indexed by topic on the pages below:
+
+.. toctree::
+   :maxdepth: 1
+   :titlesonly:
+   :glob:
+
+   topic/*
```

Comment:
Nothing is getting generated in the [rendered toctree](https://pep-previews--2579.org.readthedocs.build/topic/), presumably because this specifies `topic/` as a _relative_ path. You could specify the absolute path `/topic/*`, but much better to avoid duplication and just use the correct relative path:

```suggestion
   *
```

Reply:
Yep, I copied from when I'd added `topic/` to the main contents page & must've forgotten to adjust, thanks for picking it up!

A